web/crux/src/decorators/api-params.decorator.ts

```diff
@@ -0,0 +1,5 @@
+import { SetMetadata } from '@nestjs/common'
+
+const UuidParams = (uuidParams: string[]) => SetMetadata('uuidParams', uuidParams)
```

Comment:
Use /variable arguments/ so you can get rid of the `[]` in the annotations: `@UuidParams([PARAM_PRODUCT_ID, PARAM_VERSION_ID])` -> `@UuidParams(PARAM_PRODUCT_ID, PARAM_VERSION_ID)`
const UuidParams = (...uuidParams: string[]) => SetMetadata('uuidParams', ...uuidParams)

Reply:
Why better this solution instead of the existing one?